Makefile

```diff
@@ -105,6 +105,10 @@ else
 
 pytest: all
 	@echo "=== Running python tests ==="
+ifeq ($(shell which pytest-3),)
+	@echo "No pytest-3 found in system. Try 'apt-get install python3-pytest' or try 'which pytest-3'"
+	@echo "0 tests passed"
```

Comment:
can we make this say "0 Python tests run" instead?

Reply:
Yes, sure